ggml/src/ggml-cann.cpp

```diff
@@ -1670,10 +1670,6 @@ GGML_CALL static bool ggml_backend_cann_supports_op(ggml_backend_t backend,
                     // TODO: fix me
                     // Current groupsize should not be greater than k-1 in
                     // aclnnWeightQuantBatchMatmulV2GetWorkspaceSize().
```

Comment:
Is this comment still relevant after removing the respective code block?

Reply:
Error about this comment has not been resolved yet, keep TODO to resolve later.